specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2/backuprestore.json

```diff
@@ -174,7 +174,7 @@
         "tags": [
           "FullRestore"
         ],
-        "operationId": "FullRestoreStatus",
+        "operationId": "RestoreStatus",
         "description": "Returns the status of full restore operation",
```

Comment:
`Returns the status of full restore operation` => `Returns the status of restore operation`